algorithmic_efficiency/workloads/wmt/wmt_pytorch/models.py

```diff
@@ -106,7 +106,11 @@ def __init__(self,
                nlayers: int = 6,
                dropout_rate: Optional[float] = 0.1,
                attention_dropout_rate: Optional[float] = 0.1,
-               layer_norm_eps: float = 1e-6):
+               activation: Union[str, Callable[[Tensor], Tensor]] = F.relu,
+               glu: bool = False,
+               layer_norm_eps: float = 1e-6,
+               attention_temp: float = 1.0,
+               norm_first: bool = True):
```

Comment:
Same nit as above; can we change 'norm_fist' -> 'pre_ln'

Reply:
Already renamed everywhere in the repo.